src/Symfony/Bundle/FrameworkBundle/Resources/config/serializer.xml

```diff
@@ -26,6 +26,12 @@
             <tag name="serializer.normalizer" priority="-1000" />
         </service>
 
+        <!-- Json Normalizer -->
+        <service id="serializer.normalizer.json" class="Symfony\Component\Serializer\Normalizer\JsonSerializableNormalizer" public="false">
```

Comment:
We can also add something like `"conflict": {"symfony/serializer": "<3.1"}` in the `composer.json` of FrameworkBundle.